docs/lab/workflow/create-test.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Creating a Lab Test (Beta)"
+---
+
+Vital provides a way to create your own lab tests. These are made up of:
+
+1. One or more marker IDs.
+2. The lab ID.
+3. The collection method: `testkit`, `walk_in_test`, or `at_home_phlebotomy`.
+4. The sample type: `dried_blood_spot` or `serum`.
+
+Once a Lab Test is created and approved, you can use it when making a order.
+
+<Warning>
+  When you create a Lab Test, it is not immediately available. We need to
+  validate and approve it on our side, before you can use it when making orders.
+</Warning>
```

Comment:
```suggestion
<Warning>
  A Lab Test is not immediately available to order upon creation through the API. It would
  remain unactivated until it has been reviewed by Vital.
</Warning>
```

consolidation

Reply:
A lab test is not ordered. A lab test is associated with an order. A Lab test is a panel (i.e. a bundle of markers to be tested).